npm/rake-dist.js

```diff
@@ -0,0 +1,49 @@
+var child_process = require('child_process');
+var fs = require('fs');
+
+var Log = require('./log.js');
+var Build = require('./build.js');
+var build = new Build();
+var log = new Log();
+
+if (!fs.existsSync('build')) {
+  fs.mkdirSync('build');
+}
+
+var opalCompile = function(require, output) {
+  var stdout = child_process.execSync('opal --compile -I lib -I asciidoctor -g asciidoctor -r ' + require + ' --dynamic-require ignore --no-opal --no-exit > build/' + output);
```

Comment:
Perhaps move the opal options (e.g., `--dynamic-require ignore --no-opal --no-exit` and maybe `--compile`) to a global variable so we can easily tweak them?